components/notion/actions/add-content-to-page-or-block/add-content-to-page-or-block.js

```diff
@@ -0,0 +1,31 @@
+const notion = require("../../notion.app");
+
+module.exports = {
+  key: "notion-add-content-to-page-or-block",
+  name: "Add Content To Page Or Block",
+  description:
+    "Adds content to a specified page or block. In Notion, content is added via [blocks](https://developers.notion.com/reference/block), which can be headings, paragraphs, lists to dos, etc.",
+  version: "0.0.4",
+  type: "action",
+  props: {
+    notion,
+    blockId: {
+      type: "string",
```

Comment:
Perhaps the user could get the blockId from a previous steps in the workflow.

There is no direct way to retrieve the full set of blocks for Notion. This is what can be done.

1. Retrieve all databases
2. For each database, retrieve all pages.
3. For each page, recursively retrieve all blocks. 
Note from the Notion API: _In order to receive a complete representation of a block, you may need to recursively retrieve the block children of child blocks._

I am not convinced on going this way. 
I would discard the option to have an additional prop for pageID that would be parent to blocks, because in PD atm we can't dynamically load options of a prop given the values selected in the previous one.
Thoughts?

Reply:
@sergioeliot2039 nah, I think that's too intense and maybe not even useful? I guess lets not do an options() for blockId and see how (and if) it gets used in real life.